index.html

```diff
@@ -298,44 +292,49 @@ <h3 class="contributor__subtitle">
+                                            </div>
+                                            <div class="maintainers__contacts-item maintainers__contacts-item--twitter">
+                                                <a class="maintainers__contacts-link" href="https://twitter.com/Flyingmana" target="_blank">@Flyingmana</a>
                                             </div>
                                             <div class="maintainers__contacts-item maintainers__contacts-item--location">
-                                                <span class="maintainers__contacts-text">Lake City, FL, US</span>
+                                                <span class="maintainers__contacts-text">Germany - Berlin</span>
+                                            </div>
+                                            <div class="maintainers__contacts-item maintainers__contacts-item--email">
+                                                <a class="maintainers__contacts-link maintainers__contacts-link--limit" href="mailto:flyingmana@googlemail.com" title="flyingmana@googlemail.com" target="_blank">flyingmana@googlemail.com</a>
+                                            </div>
+                                            <div class="maintainers__contacts-item maintainers__contacts-item--link">
+                                                <a class="maintainers__contacts-link maintainers__contacts-link--limit" href="http://flyingmana.name" title="http://flyingmana.name" target="_blank">http://flyingmana.name</a>
                                             </div>
                                         </div>
                                     </div>
                                 </div>
-    
+                                
                                 <div class="maintainers__person">
                                     <div class="maintainers__view">
                                         <img class="maintainers__image" src="/assets/images/maintainers/david.png" />
-                                        <div class="maintainers__name">
-                                            David Robinson
-                                        </div>
+                                        <div class="maintainers__name">David Robinson</div>
                                     </div>
     
                                     <div class="maintainers__more-info">
                                         <div class="maintainers__stats">
                                             <div class="maintainers__stats-item">
                                                 <img class="maintainers__stats-icon" src="/assets/svg/commit.svg" />
-                                                <span class="maintainers__stats-text">53</span>
+                                                <span class="maintainers__stats-text">31</span>
```

Comment:
why did it go down?

Reply:
https://github.com/OpenMage/magento-lts/graphs/contributors

Reply:
there is also https://github.com/OpenMage/magento-mirror/graphs/contributors and https://github.com/OpenMage/OpenMage.github.io/graphs/contributors

Reply:
ahh, also I think some of these were in the older branches which now got deleted 🙄

Reply:
it becomes very difficult to sum all of the things... personally I'd even remove that data.

Reply:
reverted to 53